lib/promoted/ruby/client.rb

```diff
@@ -91,16 +91,16 @@ def deliver args, headers={}
           only_log = delivery_request_builder.only_log != nil ? delivery_request_builder.only_log : @default_only_log
           if !only_log
             cohort_membership_to_log = delivery_request_builder.new_cohort_membership_to_log
-          end
+
+            if should_apply_treatment(cohort_membership_to_log)
```

Comment:
side-note - When first reviewing the code, I noticed this difference with the TS version but figured it was on purpose.  I should have more thoroughly reviewed it.

Reply:
Nope (although I think it's technically pretty harmless).